.tekton/build-service-push.yaml

```diff
@@ -231,6 +290,47 @@ spec:
         operator: in
         values:
         - "true"
+    - name: build-sealights-container
```

Comment:
If I understand it correctly, the sealights container is only needed for e2e-tests. Do we need to build it in the push pipeline as well?

Reply:
yes, we want to run periodic jobs against main branch. The reason: Sealights to make test selection in PR and coverage we need to run periodic jobs against main branch 💀

Reply:
Ah, I see. Well, I hope it will be worth it overall :smile: